engine/frontend/src/components/Services.js

```diff
@@ -0,0 +1,146 @@
+        <div className='flex flex-col h-screen space-y-5 bg-slate-800'>
+            <div className="text-3xl text-center mt-5 mb-3 text-white"> 
+                Environments 
+            </div>
+            <div className="flex flex-col space-y-4 p-2 overflow-auto">
+                 {renderEnclaves(enclaves, handleClick)}
+            </div> 
+            {/* <div className="flex text-3xl justify-center py-10 text-white border-t-4"> {username} </div> */}
+     </div>
+    )
+}
+
+const Services = () => {
+    const navigate = useNavigate();
+    
+    const params = useParams();
+    const {name} = params;
+
+    const [services, setServices] = useState([])
+    const [fileArtifacts, setFileArtifacts] = useState([])
+
+    const {state} = useLocation();
+    const {enclaves} =  state;
+    
+    useEffect(() => {
+        const fetch = async () => {
+            //const response = await axios.get(`http://localhost:5050/enclaves/${name}/services`)
+            //setFileArtifacts(response.data.body.fileArtifacts)
+            const selected = enclaves.filter(enclave => enclave.name === name);
+            const services = await getEnclaveInformation(selected[0].apiClient);
+            setServices(services)
+        } 
+        fetch()
+    }, [name])
+
+    const handleLeftPanelClick = (enclaveName) => {
+        navigate(`/enclaves/${enclaveName}`, {state: {enclaves}, replace:true})
+    }
+
+    const handleServiceClick = (service, services) => {
+        navigate(`/enclaves/${name}/services/${service.uuid}`, {state: {services, selected: service}})
+    }
+
+    return (
+        <div className="grid grid-cols-6 h-full w-full">
+            <div className="bg-slate-800"> 
+                <LeftPanel navigate={navigate} enclaves={enclaves} handleClick={handleLeftPanelClick}/> 
+            </div>
+            <div className="col-span-5 gap-5 h-full space-y-5">
+                <div className="flex flex-col space-y-2 overflow-auto text-4xl text-center justify-center pt-2">
+                    {name}
+                </div>
+                <div className="flex flex-col space-y-2 h-1/2 border-8">
+                    <div className="space-y-2 text-2xl text-center justify-center pt-2">
+                        Services
+                    </div>
+                    <div className="overflow-auto">
+                        {renderServices(services, handleServiceClick)}
+                    </div>
+                </div>  
+                <div className="flex flex-col space-y-2 overflow-auto h-1/3 border-8">
+                    <div className="flex flex-col space-y-2 overflow-auto text-2xl text-center justify-center pt-2">
+                        Files Artifacts
+                    </div>
+                    <div className="overflow-auto">
+                        {renderFileArtifacts(fileArtifacts)}
+                    </div>
+                </div>  
+            </div>
+            {/* <div className="col-span-5 grid grid-rows-6">
```

Comment:
reminder  for a next PR, do not forget to delete the commented code

Reply:
i purposely left this commented code; because i am going uncomment them and add these things functionalities to UI.